target/linux/bcm53xx/base-files/etc/board.d/01_leds

```diff
@@ -5,6 +5,13 @@
 board_config_update
 
 case "$(board_name)" in
+linksys,panamera)
+	ucidef_set_led_usbport "usb2" "USB 2.0" "bcm53xx:green:usb2" "usb1-port2" "usb2-port2"
+	ucidef_set_led_usbport "usb3" "USB 3.0" "bcm53xx:green:usb3" "usb1-port1" "usb2-port1" "usb4-port1"
+	ucidef_set_led_netdev "wifi-enabled" "Wireless5G1" "bcm53xx:white:wifi-enabled" "wlan0" "link"
```

Comment:
Hmm, if this has multiple WiFi interfaces and this is only linked to one, it might be better to remove this line entirely and leave the configuration to the user.